### PR TITLE
fix: fixes radius issue

### PR DESCRIPTION
**Pull request by BhuvaneshPatil**
### What change does this PR introduce?

Changes styling for SegmentedControl input in Edit Email Template page
#2389 

### Why was this change needed?

Having rounded border makes UI more consisted

### Other information (Screenshots)

Dark Mode - 

image

image

Light Mode - 

image

image

### CODE REVIEW

File: apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx [hunk 1]
```diff
@@ -85,13 +85,14 @@ export function EmailMessagesCards({ index, isIntegrationActive }: { index: numb
                     width: '70px',
                   },
                   active: {
-                    background: theme.colorScheme === 'dark' ? colors.white : colors.B98,
+                    background: theme.colorScheme === 'dark' ? colors.B40 : colors.B98,
                     borderRadius: '30px',
                   },
                   labelActive: {
                     color: `${colors.B40} !important`,
```
Comment (LetItRock) [inline apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx:92]: shouldn't we apply similar color styles here as in the preview switch?

Comment (BhuvaneshPatil) [reply apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx:92]: can you please provide more details, I am not getting

Comment (p-fernandez) [reply apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx:92]: @BhuvaneshPatil what he meant is that in line 92 you should apply the same change as in line 29 of `apps/web/src/components/templates/EditorPreviewSwitch.tsx`.

Comment (BhuvaneshPatil) [reply apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx:92]: Thank @p-fernandez , I'll update and push the new changes

**Comment by jainpawan21** [comment]
https://user-images.githubusercontent.com/39362422/210101858-0b9201bb-efea-4983-bc30-e16b73791b03.mov

I think it is fixed already.

I tested in latest next branch and prod web.novu.co

**Review by scopsy — APPROVED** [review]

**Pull request merged**